elements/rh-avatar/rh-avatar.ts

```diff
@@ -9,6 +9,7 @@ import { ifDefined } from 'lit/directives/if-defined.js';
 import { colorContextConsumer, colorContextProvider } from '../../lib/context/color.js';
 import type { ColorPalette, ColorTheme } from '../../lib/context/color.js';
 import { CssVariableController } from '@patternfly/pfe-core/controllers/css-variable-controller.js';
+import { tokens } from '@rhds/tokens';
```

Comment:
we can't use this in client-facing code, it'll drag in the entire map, which is just too much js to justify the convenience of `.get`.

Reply:
can you set it in the css the query the computedStyle at runtime?